lib/src/controls/navigation/tree_view.dart

```diff
@@ -533,7 +645,7 @@ class _TreeViewItem extends StatelessWidget {
                       ),
                     ),
                   Padding(
-                    padding: const EdgeInsetsDirectional.only(start: 18.0),
+                    padding: const EdgeInsetsDirectional.only(start: 10.0),
```

Comment:
why was this value changed?

Reply:
Good point. Before I had added the `_anyExpandableSiblings` concept it was always adding an extra 8 pixels of horizontal padding for every item, even for siblings who were all not expandable, so this change was to offset that in the other direction. Now that the `_anyExpandableSiblings` value is calculated, it only needs to add the extra padding if there is indeed one sibling who is expandable within that given part of the tree. So this can be changed back to what it was.